pages/_app.js

```diff
@@ -1,16 +1,91 @@
 import App from 'next/app';
 import Head from 'next/head';
-import React from 'react';
+import React, { useEffect } from 'react';
 import { GlobalStyle } from '@catho/quantum';
+import Image from 'next/image'
+import logoImage from '../static/imgs/cathoLogo.png'
+import backgroundImg from '../static/imgs/background-image.png'
```

Comment:
what is the pattern for the image names?
one is img another os Image. 
How about of choose only one patter?

Reply:
The second isn't being used, I happened to forget to remove.